src/core/tap-snapshots/test/test-base.ts.test.cjs

```diff
@@ -51,26 +51,6 @@ TAP version 14
     ok 1 - child # time={TIME}
     
     not ok 2 - cannot create subtest after parent promise resolves
-      ---
-      stack: |
-        Minimal.<anonymous> (test/test-base.ts:1664:13)
-      at:
-        fileName: test/test-base.ts
-        lineNumber: ##
-        columnNumber: ##
-        typeName: Minimal
-        methodName: <anonymous>
-        functionName: Minimal.<anonymous>
-      test: child
-      source: |2
-            if (!subtest) throw new Error('did not get subtest')
-            await p
-            subtest.test('this should not work', async () => {})
-        ------------^
-          })
-          tb.end()
-      ...
-    
```

Comment:
This test was wrong from the beginning, since we're setting an explicit `diagnostics: false` in the parent test

Reply:
Love it when the snapshots show that it wasn't actually working! It's like getting your repro for free :)